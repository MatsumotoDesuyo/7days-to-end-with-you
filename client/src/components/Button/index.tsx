import React from 'react';
import './index.scss';
import './index.css';

export type ButtonProps = {
  label: string;
};

// eslint-disable-next-line @typescript-eslint/no-unused-vars,no-unused-vars
const style = {
  backgroundColor: '#007bff',
  color: '#fff',
  border: '0',
  borderRadius: '.25rem',
  fontSize: '1rem',
};

const Button: React.FC<ButtonProps> = ({ label }) => (
  <button type="button" className="scsstst csstst">
    hage{label}
  </button>
);

export default Button;
