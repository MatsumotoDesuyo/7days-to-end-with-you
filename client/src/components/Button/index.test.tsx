import React from 'react';
import renderer from 'react-test-renderer';
import Button from '.';

test('test test', () => {
  const component = renderer.create(<Button label="fuga" />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
