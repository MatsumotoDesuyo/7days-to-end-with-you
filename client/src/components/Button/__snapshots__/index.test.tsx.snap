// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test test 1`] = `
<button
  className="scsstst csstst"
  type="button"
>
  hage
  fuga
</button>
`;
